solax/inverter.py

```diff
@@ -65,18 +69,15 @@ async def make_request(cls, host, port, pwd="", headers=None) -> InverterRespons
         raise NotImplementedError()
 
     @classmethod
-    def sensor_map(cls) -> Dict[str, Tuple[int, Measurement]]:
+    def sensor_map(cls) -> Dict[str, Tuple[SensorIndexSpec, Measurement]]:
```

Comment:
another problem: home-assistant relies on the structure of the response from `sensor_map`.

https://github.com/home-assistant/core/blob/dev/homeassistant/components/solax/sensor.py#L42

Reply:
But we have to change it anyway due to this: https://github.com/squishykid/solax/pull/70#issue-1212529672